subgraphs/morpho/src/helpers.ts

```diff
@@ -950,16 +1004,61 @@ function snapshotPosition(position: Position, event: ethereum.Event): void {
 export function updateProtocolPosition(
   protocol: LendingProtocol,
   market: Market,
-  morphoPositions: MorphoPositions
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why do we want to keep this parameter if it isn't being used?

Reply:
My bad, I will remove it